src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -364,10 +364,28 @@ public virtual string CustomMethod
         #region Output
 
         /// <summary>
-        /// Gets or sets the OutFile property.
+        /// Gets or sets the Path property.
         /// </summary>
         [Parameter]
-        public virtual string OutFile { get; set; }
+        [Alias("OutFile")]
+        public virtual string Path
+        {
+            get => _path;
+            set => _path = value;
+        }
+
+        private string _path;
+
+        /// <summary>
+        /// Gets or sets the LiteralPath property.
+        /// </summary>
+        [Parameter]
+        [Alias("LP")]
```

Comment:
all the other cmdlets which provide `LiteralPath` also have a `PSPath` alias. Should that be done here?

Reply:
It seems the PSPath is mainly used for pipeline parameter binding. Because the parameter doesn't get arguments from pipeline I removed the alias.